apps/src/p5lab/redux/textConsole.js

```diff
@@ -7,8 +7,8 @@ export default function textConsole(state, action) {
       return [];
     case ADD_MESSAGE:
       return [
-        // 1000 items is an arbitrary limit. Change as needed.
-        ...state.slice(0, 999),
+        // 1000 is an arbitrary limit - the last 1000 text console statements will be displayed
+        ...state.slice(-999),
```

Comment:
Should we store this 999 in constants as well? (or store it as a 1000 and convert it to -999 here so the actual limit value is clearer)

Reply:
Yes - this makes sense! I updated in the prior PR that first made this change. https://github.com/code-dot-org/code-dot-org/pull/47738